splash/har/qt.py

```diff
@@ -114,7 +119,11 @@ def reply2har(reply, include_content=False, binary_content=False):
 
     status_text = reply.attribute(QNetworkRequest.HttpReasonPhraseAttribute)
     if status_text is not None:
-        res["statusText"] = bytes(status_text).decode('latin1')
+        try:
+            res["statusText"] = bytes(status_text, 'latin1')
+        except TypeError:
+            res["statusText"] = bytes(status_text).decode('latin1')
```

Comment:
This code is not clear: what should be the type of `res['statusText']` - bytes or unicode? If it should be bytes then the second branch is wrong, if it should be unicode the first branch is wrong. I think in most cases the type of a value should be the same in Python 3 and Python 2.